Week 1 - C/mario-more/mario.c

```diff
@@ -0,0 +1,38 @@
+#include <cs50.h>
+#include <stdio.h>
+
+void printHashtag(int n)
```

Comment:
actually, I think these 2 functions are too similar, does it make sense to combine them?